_pages/get-involved/become-climbing-leader.md

```diff
@@ -22,7 +22,9 @@ Climbing is inherently dangerous. We want to make sure our leaders are prepared
 
 ### How do I become a leader?
 
-Please [submit an application](https://mitoc-trips.mit.edu/climbing/leaders/apply/). This is an informal "resume" summarizing both climbing and outdoor leadership experience. We also ask for 2 recommendations from current climbing leaders. Tell your recommenders what rating you are applying for and that they should email climbing-chair@mit.edu with their recommendations. Applicants will be assigned a "Leader" or "Co-leader" rating based on the resume. Co-leaders will need to co-lead a trip with a full leader, but can be considered for full-leader status after they have co- led a trip or a few.
+Due to the inherit risk involved in climbing and the variety of skills required, MITOC uses a leader rating system with different categories for different styles of climbing. MITOC leaders should be experienced and comfortable in the styles that they are rated for.
+
```

Comment:
Submit mentioned here twice. Otherwise, looks great!

Reply:
Thanks for catching that- I'll fix it now